Build failure for samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy on rv32m1_vega_ri5cy

**Describe the bug**
Build failure for samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy on rv32m1_vega_ri5cy

**Impact**
broken shippable

**Screenshots or console output**
```
[184/188] Generating linker_pass_final.cmd
[185/188] Generating isr_tables.c
FAILED: cd /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/sanity-out/rv32m1_vega_ri5cy/samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy/zephyr && /opt/sdk/zephyr-sdk-0.10.3/riscv64-zephyr-elf/bin/riscv64-zephyr-elf-objcopy --input-target=elf32-littleriscv --output-target=binary --only-section=.intList /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/sanity-out/rv32m1_vega_ri5cy/samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy/zephyr/zephyr_prebuilt.elf isrList.bin && /usr/bin/python3 /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/common/gen_isr_tables.py --output-source isr_tables.c --kernel /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/sanity-out/rv32m1_vega_ri5cy/samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy/zephyr/zephyr_prebuilt.elf --intlist isrList.bin   --sw-isr-table
gen_isr_tables.py: multiple registrations at table_index 25 for irq 25 (0x19)
Traceback (most recent call last):
  File "/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/common/gen_isr_tables.py", line 314, in <module>
    main()
  File "/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/common/gen_isr_tables.py", line 306, in main
    error("multiple registrations at table_index %d for irq %d (0x%x)" % (table_index, irq, irq))
  File "/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/common/gen_isr_tables.py", line 36, in error
    raise Exception()
Exception
ninja: build stopped: subcommand failed.
```

--- a/subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/ll_irqs.h
+++ b/subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/ll_irqs.h
@@ -13,15 +13,15 @@
  * LPTMR1 -> INTMUX_CH2. We expect it to be the only IRQ source for this channel
  * We'll use the INTMUX ISR for channel 2 instead of LPTMR1 ISR
  */
-#define LL_RTC0_IRQn_1st_lvl DT_INST_2_OPENISA_RV32M1_INTMUX_CH_IRQ_0
-#define LL_RTC0_IRQn_2nd_lvl DT_INST_1_OPENISA_RV32M1_LPTMR_IRQ_0
+#define LL_RTC0_IRQn_1st_lvl DT_OPENISA_RV32M1_INTMUX_CH_4004F080_IRQ_0
+#define LL_RTC0_IRQn_2nd_lvl DT_OPENISA_RV32M1_LPTMR_40033000_IRQ_0
 
 /*
  * radio -> INTMUX_CH3. We expect it to be the only IRQ source for this channel
  * We'll use the INTMUX ISR for channel 3 instead of radio ISR
  */
-#define LL_RADIO_IRQn_1st_lvl DT_INST_3_OPENISA_RV32M1_INTMUX_CH_IRQ_0
-#define LL_RADIO_IRQn_2nd_lvl DT_INST_0_OPENISA_RV32M1_GENFSK_IRQ_0
+#define LL_RADIO_IRQn_1st_lvl DT_OPENISA_RV32M1_INTMUX_CH_4004F0C0_IRQ_0
+#define LL_RADIO_IRQn_2nd_lvl DT_OPENISA_RV32M1_GENFSK_41033000_IRQ_0
 
 #define LL_RTC0_IRQn LL_RTC0_IRQn_1st_lvl
 #define LL_RADIO_IRQn LL_RADIO_IRQn_1st_lvl

